flocker/node/agents/test/test_blockdevice.py

```diff
@@ -1217,9 +1233,12 @@ def test_listed_volume_attributes(self):
             size=REALISTIC_BLOCKDEVICE_SIZE
         )
         [listed_volume] = self.api.list_volumes()
+        volume_size_GiB = int(Byte(listed_volume.size).to_GiB().value)
+        create_size_GiB = int(Byte(REALISTIC_BLOCKDEVICE_SIZE).to_GiB().value)
+
         self.assertEqual(
-            (expected_dataset_id, REALISTIC_BLOCKDEVICE_SIZE),
-            (listed_volume.dataset_id, listed_volume.size)
+            (expected_dataset_id, create_size_GiB),
+            (listed_volume.dataset_id, volume_size_GiB)
```

Comment:
A link to FLOC-1874 would probably be appropriate here.  I'd really like these tests to make a more precise assertion than one truncated down to GiB.